src/features/duplicates/components/FieldSettings/index.tsx

```diff
@@ -11,28 +13,51 @@ interface FieldSettingsProps {
 
 const FieldSettings: FC<FieldSettingsProps> = ({ fieldValues, onChange }) => {
   const theme = useTheme();
+  const messages = useMessages(messageIds);
 
   return (
-    <Box
-      border={1}
-      sx={{ borderColor: theme.palette.grey[300], borderRadius: 2 }}
-    >
-      {Object.entries(fieldValues).map((entry, index) => {
-        const [field, values] = entry;
-        return (
-          <>
-            {index !== 0 && <Divider />}
-            <FieldSettingsRow
-              key={index}
-              field={field as NATIVE_PERSON_FIELDS}
-              onChange={(selectedValue: string) =>
-                onChange(field as NATIVE_PERSON_FIELDS, selectedValue)
-              }
-              values={values}
-            />
-          </>
-        );
-      })}
+    <Box>
+      <Typography variant="h6">
+        {messages.modal.fieldSettings.title()}
+      </Typography>
+      <Box
+        alignItems="center"
+        display="flex"
+        justifyContent="space-between"
+        padding={1}
+      >
+        <Box flexDirection="column" marginRight={2} width="50%">
+          <Typography color={theme.palette.grey[500]} variant="subtitle2">
+            {messages.modal.fieldSettings.field()}
+          </Typography>
+        </Box>
+        <Box flexDirection="column" marginRight={2} width="50%">
+          <Typography color={theme.palette.grey[500]} variant="subtitle2">
+            {messages.modal.fieldSettings.data()}
+          </Typography>
+        </Box>
+      </Box>
+      <Box
+        border={1}
+        sx={{ borderColor: theme.palette.grey[300], borderRadius: 2 }}
+      >
+        {Object.entries(fieldValues).map((entry, index) => {
+          const [field, values] = entry;
+          return (
+            <>
+              {index !== 0 && <Divider />}
+              <FieldSettingsRow
+                key={index}
```

Comment:
I think `field` should be the key. The purpose of a key is to tell React which element is which when there's a series of otherwise identical elements. This is valuable to React in order to know whether to update an existing DOM element or add a new one.

Using indices as keys is generally discouraged because it doesn't really uniquely identify the element to React. If the array changes, the indices also change.

If you make `field` the key I believe the `index` variable becomes unnecessary which is always a welcome effect.